### PR TITLE
Proc 897 subscription update enpoint

### PR DESCRIPTION
**Pull request by ibudasov**

### CODE REVIEW

File: source/reference/v2/subscriptions-api/update-subscription.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+   * - ``startDate``
+
+       .. type:: date
+          :required: true
+
+     - The start date of the subscription in ``YYYY-MM-DD`` format. This is the first day on which your customer will be charged. Should always be in the future
+
+   * - ``description``
+
+       .. type:: string
```
Comment (willemstuursma) [inline source/reference/v2/subscriptions-api/update-subscription.rst]: Should be `date` . https://docs.mollie.com/guides/common-data-types#date

File: source/reference/v2/subscriptions-api/update-subscription.rst [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+Replace ``customerId`` in the endpoint URL by the customer's ID, and replace ``id`` by the subscription's ID. For
+example: ``/v2/customers/cst_5a2pPrwaWy/subscriptions/sub_8EjeBVgtEn``.
+
+.. list-table::
+   :widths: auto
+
+   * - ``amount``
+
+       .. type:: object
+          :required: true
```
Comment (willemstuursma) [inline source/reference/v2/subscriptions-api/update-subscription.rst]: They are all required? Or is is possible just to update the amount?

**Comment by willemstuursma** [comment]
Should we support patch for other API endpoints that update things?

Could you please add an entry to the v2 changelog.

File: source/reference/v2/subscriptions-api/update-subscription.rst [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        },
+        "description": "Mollie Recurring subscription",
+        "method": null,
+        "times": 42,
+        "interval": "15 days",
+        "startDate": "2018-12-12",
+        "webhookUrl": "https://example.org/webhook",
+        "_links": {
+            "self": {
+                "href": "http://api.mollie.test/v2/customers/cst_5a2pPrwaWy/subscriptions/sub_8EjeBVgtEn",
```
Comment (martijnpolder) [inline source/reference/v2/subscriptions-api/update-subscription.rst]: Admittedly minor, but the request uses `https://api.mollie.com`, the response returns `http://api.mollie.test`

Comment (willemstuursma) [reply source/reference/v2/subscriptions-api/update-subscription.rst]: We should have some linting for that. 

Could you add a step to the `Makefile` that makes the build fail if there is `api.mollie.test` in a `.rst` file.

**Comment by willemstuursma** [comment]
Made some small copy changes.

File: source/reference/v2/subscriptions-api/update-subscription.rst [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+.. list-table::
+   :widths: auto
+
+   * - ``amount``
+
+       .. type:: object
+          :required: false
+
+     - The amount that you want to charge, e.g. ``{"currency":"EUR", "value":"100.00"}`` if you would want to the charge
```
Comment (martijnpolder) [inline source/reference/v2/subscriptions-api/update-subscription.rst]: > if you would want to *change* the charge 

I assume?

**Pull request merged**